eng/scripts/Create-Apiview-Token-Cadl.ps1

```diff
@@ -11,6 +11,7 @@ param (
 
 Set-StrictMode -Version 3
 $sparseCheckoutFile = ".git/info/sparse-checkout"
+$globalPackageJsonPath = "specification/package.json"

```

Comment:
Note this is going to change to the root of the repo soon in https://github.com/Azure/azure-rest-api-specs/pull/22735.